sale_triple_discount/models/sale_order_line.py

```diff
@@ -11,6 +11,11 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
+    discount1 = fields.Float(
+        string="Disc. 1 (%)",
+        digits="Discount",
+        default=0.0,
+    )
```

Comment:
@grindtildeath  any reason to not add this field  in the view: https://github.com/OCA/sale-workflow/blob/16.0/sale_triple_discount/views/sale_order_report.xml#L7 ?